packages/hoppscotch-app/helpers/types/RESTMethod.ts

```diff
@@ -0,0 +1,11 @@
+export type RESTMethod =
+  | "get"
+  | "post"
+  | "put"
+  | "patch"
+  | "delete"
+  | "head"
+  | "connect"
+  | "options"
+  | "trace"
+  | "custom"
```

Comment:
method param is a string for a reason ;)
"CUSTOM" is not a valid REST Method, instead when the method is set to CUSTOM, the user can edit the Method field to use a custom text. Hence, any valid string can happen.